apps/typegpu-docs/src/content/examples/rendering/3d-fish/index.html

```diff
@@ -21,19 +21,30 @@
     z-index: 1;
     background-color: rgba(0, 0, 0, 0.7);
     color: white;
-    padding: 10px;
-    border-radius: 10px;
+    padding: 0.625rem;
+    border-radius: 0.625rem;
     user-select: none;
     pointer-events: none;
     transition: opacity 0.5s;
   }
+
+  .spinner-background {
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    background-color: var(--color-gray-100);
+    z-index: 4;
+  }
 </style>
+
+<div id="spinner-background" class="spinner-background"></div>
+<div id="spinner" class="spinner"></div>
```

Comment:
put spinner as a child element of spinner-background. it just makes sense + you can then set display: none just on the background to hide both of them